frontend/views/pages/GroupDashboard.vue

```diff
@@ -2,19 +2,23 @@
 page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='groupSettings.groupName')
   template(#title='') {{ groupSettings.groupName }}
 
+  welcome(v-if='groupIncomeDistribution.length === 0')
+
   start-inviting(v-if='groupMembersCount === 1')
 
-  proposals-widget
+  div(v-else)
+    page-section(title='This months overview' v-if='!showHistory')
+      span.support-history.button.is-outlined.is-small(@click='showHistory = !showHistory') Support history
+      overview
 
-  page-section(title='This months overview')
-    overview
+    page-section(title='Support History' v-if='showHistory')
+      span.support-history.button.is-outlined.is-small(@click='showHistory = !showHistory') This month's overview
+      support-history
```

Comment:
The Dashboard should not contain this sort of logic. Imagine it as a place for pluggable widgets. The widgets are the things at the top-level, like `overview`, `support-history`, and `proposals-widget`.

Imagine also that these widgets can be dragged and re-arranged to the users liking. This is something we were/are considering doing in the future.

What defines a "widget" is basically the entire thing that is currently being represented as a card. You should also imagine that in the future we can have different themes that define widgets using different borders, i.e. the cards can be themeable. etc.

So anything within a widget, must be within the widget, and not here. Meaning, the toggle button that reads "Support History" vs "Monthly Overview" does not exist here, but within the widget itself.

So these 7 lines should be collapsed into 1 line that reads something like:

```pug
contributions-overview(v-else)
```

Then you can create a folder for that widget, and create 3 files: one file to contain and toggle between the two sub-views, and the two sub-views themselves.

Reply:
Also, on a totally separate note (somewhat irrelevant since this code is being removed, but as a reminder when it's added back in the other files), lines 11 and 15 here are missing `i18n` tags